sdk/java-sdk-spring/src/main/java/kalix/javasdk/annotations/Trigger.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk.annotations;
+
+import java.lang.annotation.*;
+
+/**
+ * This annotation is used to mark a method as a hook that will be triggered automatically uppon certain events.
+ * The method must be public and have no parameters.
+ * If the call fails, it will be retried up to the number of times specified by the maxRetries parameter.
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Trigger {
+  enum TriggerEvent {
+    UNSPECIFIED,
```

Comment:
why do we need unspecified here?

Reply:
good point, I was just folliwing the proto specification without thinking too much.. removed now, thanks